install_config/cluster_metrics.adoc

```diff
@@ -108,52 +108,62 @@ secrets:
+Before it can deploy components, the *metrics-deployer* service account must be
+granted the `edit` permission for the *openshift-infra* project:
 
-[options="nowrap"]
 ----
-$ oadm policy add-role-to-user edit system:serviceaccount:openshift-infra:metrics-deployer
+$ oadm policy add-role-to-user \
+    edit system:serviceaccount:openshift-infra:metrics-deployer
 ----
 
-[[heapster-service]]
-
+[[heapster-service-account]]
 === Heapster Service Account
 
 The Heapster component requires access to the master server to list all
 available nodes and access the `/stats` endpoint for each node. Before it can do
 this, the Heapster service account requires the `cluster-reader` permission:
 
-[options="nowrap"]
 ----
-$ oadm policy add-cluster-role-to-user cluster-reader system:serviceaccount:openshift-infra:heapster
+$ oadm policy add-cluster-role-to-user \
+    cluster-reader system:serviceaccount:openshift-infra:heapster
 ----
 
-== Metric Data Storage
+[NOTE]
+====
+The Heapster service account is created automatically during the
+link:../install_config/cluster_metrics.html#deploying-the-metrics-components[Deploying
+the Metrics Components] step.
+====
+
+[[metrics-data-storage]]
+== Metrics Data Storage
 
 You can store the metrics data to either
-link:../architecture/additional_concepts/storage.html[persistent storage]
-or to a temporary
-link:../dev_guide/volumes.html[pod volume].
+link:../architecture/additional_concepts/storage.html[persistent storage] or to
+a temporary link:../dev_guide/volumes.html[pod volume].
 
+[[metrics-persistent-storage]]
 === Persistent Storage
 
 Running OpenShift cluster metrics with persistent storage means that your
 metrics will be stored to a
 link:../architecture/additional_concepts/storage.html#persistent-volumes[persistent
 volume] and be able to survive a pod being restarted or recreated. This is ideal
-if you require your metric data to be guarded from data loss.
+if you require your metrics data to be guarded from data loss.
 
-For cluster metrics to work with persistent storage, ensure that the persistent volume has the *ReadWriteOnce* access mode. If not, the persistent volume claim will not be able to find the persistent volume, and Cassandra will fail to start.
+For cluster metrics to work with persistent storage, ensure that the persistent
+volume has the *ReadWriteOnce* access mode. If not, the persistent volume claim
+will not be able to find the persistent volume, and Cassandra will fail to
+start.
 
 To use persistent storage with the metric components, ensure that a
 link:../architecture/additional_concepts/storage.html#persistent-volumes[persistent
 volume] of sufficient size is available. The creation of
 link:../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent
 volume claims] is handled by the
-link:../install_config/cluster_metrics.html#metrics-deployer[*metrics
-deployer*].
+link:../install_config/cluster_metrics.html#metrics-deployer[Metrics Deployer].
```

Comment:
Unless it was an explicit reference to an `metrics-deployer` object (with a hyphen).